pkgs/development/node-packages/node-packages-v10.json

```diff
@@ -1,9 +1,142 @@
 [
-  "bower"
+  "alloy"
+, "asar"
+, "azure-cli"
+, "azure-functions-core-tools"
+, "bower"
+, "bower2nix"
+, "browserify"
+, "castnow"
+, "clean-css"
 , "coffee-script"
+, "coinmon"
+, "configurable-http-proxy"
+, "cordova"
+, "create-cycle-app"
+, "create-react-app"
+, "create-react-native-app"
+, "csslint"
+, "dat"
+, "dhcp"
+, "dnschain"
+, "docker-registry-server"
```

Comment:
No update in three years: https://www.npmjs.com/package/docker-registry-server
I think we have better, maintained alternatives in nixpkgs already.